crates/core_simd/tests/ops_impl/int_macros.rs

```diff
@@ -228,6 +228,28 @@ macro_rules! int_tests {
                 assert_biteq!(a, expected);
             }
 
+            #[test]
+            #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+            #[should_panic]
```

Comment:
Wasm does not support #[should_panic] tests. https://github.com/rustwasm/wasm-bindgen/issues/2286

I am personally OK with not running these tests on wasm if my collaborators do not object.

Reply:
Oh right, no unwinding in wasm.  Fine with me.

Reply:
OK - I'll wait for others to chime in.